scripts/install_via_pip.sh

```diff
@@ -50,7 +50,7 @@ fi
 
 # install pytorch nightly if asked for
 if [[ $PYTORCH_NIGHTLY == true ]]; then
-  sudo pip install torch_nightly -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
+  sudo pip install --upgrade --pre torch -f https://download.pytorch.org/whl/nightly/cpu/torch_nightly.html
```

Comment:
can you add what these changes are to your PR?

Reply:
The torch nightly build name changed, so this was actually still using an old nightly version from last August, which was still torch 1.3. I've updated this command, so it now obtains the latest nightly build.